libraries/AP_Landing/AP_Landing_Slope.cpp

```diff
@@ -346,26 +346,24 @@ int32_t AP_Landing::type_slope_get_target_airspeed_cm(void)
 
     const float land_airspeed = tecs_Controller->get_land_airspeed();
     int32_t target_airspeed_cm = aparm.airspeed_cruise_cm;
-
+    if (land_airspeed >= 0) {
+        target_airspeed_cm = land_airspeed * 100;
+    } else {
+        target_airspeed_cm = 0.5 * (aparm.airspeed_cruise_cm * 0.01 + aparm.airspeed_min);
```

Comment:
Sorry, just spotted this. Airspeed min is in m/s.

```suggestion
        target_airspeed_cm = 0.5 * (aparm.airspeed_cruise_cm + aparm.airspeed_min * 100.0);
```